### PR TITLE
[docs] - Reformat category pages

### PR DESCRIPTION
**Pull request by erinkcochran87**
This PR reformats the following category pages in the docs:

- [Guides](https://dagster-git-erin-category-pages-elementl.vercel.app/guides)
- [Concepts](https://dagster-git-erin-category-pages-elementl.vercel.app/concepts)
- [Integrations](https://dagster-git-erin-category-pages-elementl.vercel.app/integrations)

### CODE REVIEW

File: docs/content/concepts.mdx [hunk 1]
```diff
@@ -1,89 +1,241 @@
+---
+title: Concepts | Dagster Docs
+---
+
 # Concepts
 
-Dagster is a data orchestrator. It lets you define jobs in terms of the data flow between logical components called ops. These jobs can be developed locally and run anywhere.
+TODO
```
Comment (erinkcochran87) [inline docs/content/concepts.mdx:7]: TODO

File: docs/content/concepts.mdx [hunk 2]
```diff
@@ -1,89 +1,241 @@
+---
```
Comment (erinkcochran87) [inline docs/content/concepts.mdx:1]: The goal of this page is to act as a jumping-off point, so the copy should be kept high-level and brief.

Comment (sryza) [inline docs/content/concepts.mdx:1]: Huge improvement

File: docs/content/guides.mdx [hunk 3]
```diff
@@ -1,16 +1,49 @@
...
+  <ArticleListItem
+    title="Enriching with Software-defined Assets"
+    href="/guides/dagster/enriching-with-software-defined-assets"
+  ></ArticleListItem>
+  <ArticleListItem
+    title="Using the Hacker News example project"
+    href="/guides/dagster/example_project"
+  ></ArticleListItem>
+  <ArticleListItem
+    title="Re-excuting Dagster jobs"
```
Comment (sryza) [inline docs/content/guides.mdx:19]: Re-executing

File: docs/content/guides.mdx [hunk 4]
```diff
@@ -1,16 +1,49 @@
+---
+title: Guides | Dagster Docs
+---
+
 # Guides
 
-## Dagster Guides
+Learn to accomplish common tasks in Dagster.
```
Comment (sryza) [inline docs/content/guides.mdx:7]: I think losing the descriptions on this page might sacrifice more context than is worth it for the streamlining - each guide is a bird of such a different feather that seeing them in a list without additional context makes it difficult to get a handle on what's going on.

E.g. it's hard for me to imagine that someone coming to Dagster and seeing "Validating data with Dagster Type factories" would have any idea about what to do with it.

I also think several (most?) of these guides don't really fit the description "accomplish common tasks in Dagster":
- Enriching with Software-defined Assets - this is a guide on how to upgrade, not a guide on how to accomplish a task using Dagster
- Migrating to graphs, jobs, and ops - same as above
- Using the Hacker News example project - this is an overview of an example, not a task you'd accomplish

File: docs/content/integrations.mdx [hunk 5]
```diff
@@ -5,59 +5,173 @@ description: Dagster is flexible and allows incremental adoption. It provides ad
 
 # Integrations
 
-Dagster is flexible and allows incremental adoption. It provides add-on libraries to integrate with your existing tools and infrastructure.
```
Comment (sryza) [inline docs/content/integrations.mdx:8]: I think this is an improvement

File: docs/content/guides.mdx (hunk 3 above)
Comment (erinkcochran87) [reply docs/content/guides.mdx:19]: Good catch, thanks!

File: docs/content/guides.mdx (hunk 4 above)
Comment (erinkcochran87) [reply docs/content/guides.mdx]: That's a good point. This page was really tricky - I definitely didn't love it when I pushed this up the first time.

I took another shot at this. It doesn't use the two-column list like the other pages do, however - I opted for a single so summaries could be added without everything running together. At least with this approach (versus the original table), the elements are consistent from page to page, even if the columns are different.

**Review by sryza — APPROVED** [review]
I like it!

**Pull request merged**